portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx

```diff
@@ -14,18 +14,18 @@ updated: ""
 ## Intro
 
 In this guide we'll create an **auction** on our own **marketplace** with the TypeScript SDK! In a **marketplace** you can list your tokens, like NFT's, for sale. Opensea is an example of a **marketplace**.
-Did you know you can do both! We will list an NFT from the NFT Collection module. If you don't have any NFTs minted, check out this guide to mint some nfts.
+Did you know you can do both! We will list an NFT from the NFT Collection contract. If you don't have any NFTs minted, check out this guide to mint some nfts.
 
-By the way we also need to connect a wallet. So if you don't have a wallet, make sure to sign up with MetaMask or CoinWallet.
+By the way we also need to connect a wallet. So if you don't have a wallet, make sure to sign up with Coinbase, MetaMask or CoinWallet.
 If don't have a wallet, [you can follow this guide to create one](/guides/create-a-metamask-wallet).
 
 Alright without further ado, let's create our **marketplace**!
 
 ## Dashboard setup
 
-The first thing we need to do is head over to the [dashboard](https://thirdweb.com/dashboard) and create a project and a **Marketplace** module. if you don't know how to do this, check out [this guide](/learn/introduction).
+The first thing we need to do is head over to the [dashboard](https://thirdweb.com/dashboard) and create a **Marketplace** contract. if you don't know how to do this, check out [this guide](/learn/introduction).
 
-Once you've created the module, it's time to list our first NFT, but we don't need the dashboard for that! I mean we could, but why use GUI if we can use TypeScript!?
+Once you've created the contract, it's time to list our first NFT, but we don't need the dashboard for that! I mean we could, but why use GUI if we can use TypeScript!?
 
 ![](/assets/portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk/17-marketplace.png)
```

Comment:
All the screenshots need to be updated with the new nightly dashboard

Reply:
done